### ISSUE TITLE
OCGS trying to send out emails to meeting attendees   ****.****@***.***.ogcs

### ISSUE THREAD

**Issue by almartel** [phw198/OutlookGoogleCalendarSync#1155]
OGCS v 2.8.0.0

I use OCGS to sync an outlook calendar to my gmail calendar ie events from outlook are copied to the gmail calendar

I periodically get a batch of mail delivery errors on my gmail account: OCGS appears to be sending out emails to other attendees on meetings that are synced - the bounced emails all have .ogcs tagged onto the end of the address which is why I am assuming this is the source of the problem. I can't see any content in these emails (not sure if it gets stripped off or if there never was a message attached!)

**Comment by phw198** [comment]
Working as designed and protecting Google from spamming your meeting attendees: [the wiki covers this](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs#why-are-my-meeting-attendees-getting-notified-of-updates-to-events-in-google).

**Comment by WarningU2** [comment]
I recently had this happen too where OGCS was sending out 100's of emails which came back as invalid that was flooding my inbox .   I had to get up in the night and shutdown OGCS and clean up the messages.   Restarting OGCS normal operations resumed.   I don't think this is working as designed.   :)   Sorry I don't have a debug log ... it was a question of stopping the recursive messages immediately or watching my PC crash.

**Comment by phw198** [comment]
OGCS doesn't send any emails at all. OGCS syncing attendees and meeting alterations to Googel calendar may cause _Google_ to spam those attendees, but that's why there's the "Cloak email" feature. You have two options, 1) don't sync attendees, then Google can't attempt to spam them; or 2) don't cloak attendee emails, then the spam will be delivered and you won't get a bounce.

**Comment by WarningU2** [comment]
Ok so I've disabled the emails you don't send :) by setting attendees to blank.   The vernacular not withstanding, 99.9% of the syncing attendees occurs without issue but just in the last few months I've noted the emails from ** OGCS showing up in my inbox once in a while.   Now turned off that prevents that from happening.   Great.   But there wasn't an issue until a couple of releases ago.   

![image](https://user-images.githubusercontent.com/44507492/121820287-05226900-cc60-11eb-95dd-dc06c705a6d9.png)

**Comment by phw198** [comment]
OK, but to reiterate, this is not something I have any control over - it's a Google "feature" which I have noted in the wiki already linked.

Be aware that unchecking attendees in OGCS will only stop further syncing of attendees - so if you e.g. delete an item on your mobile calendar app that has previously synced attendees, Google will still email them whether you opt for an update to be sent to them or not.

**Comment by WarningU2** [comment]
Paul sorry but your tactile refusal to admit that something is wrong is a little disturbing to say the least.  I too will reiterate ....  There was nothing wrong with the syncing of attendees for several years until a few months ago.   Hopefully unchecking the box will resolve this issue at least for me and others that started this thread.

**Comment by jesseelliott** [comment]
> Paul sorry but your tactile refusal to admit that something is wrong is a little disturbing to say the least. I too will reiterate .... There was nothing wrong with the syncing of attendees for several years until a few months ago. Hopefully unchecking the box will resolve this issue at least for me and others that started this thread.

Don't you think it's _possible_ that Google changed something in the last months??

**Comment by phw198** [comment]
I'll explain once more:
* Nothing in OGCS has changed recently regarding how events are updated in Google
* The [API documentation](https://developers.google.com/calendar/api/v3/reference/events/insert) has an option for sending alerts - OGCS has **never** set this to anything other than the default for `sendNotifications` or `sendUpdates`, which is `false` / `none` respectively.
* The issue has been around since before the start of OGCS, affecting Google's own calendar sync app - see the [wiki](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs#why-are-my-meeting-attendees-getting-notified-of-updates-to-events-in-google)
* I have already spent time mitigating Google's issue by "cloaking" attendee emails - the choice is now _yours_ on whether you potentially spam people, or receive bounce backs
* The fact that it "randomly" hits different users at different times, I would suggest supports the reasoning the bug is Google's side
* The issue can be consistently replicated **entirely** outside of OGCS
     * Browser to calendar.google.com and create a Google event with an attendee with an email address that doesn't exist (eg ending `.ogcs`)
     * Save it and open your mobile phone Google calendar app (I tested this on Android)
     * Delete the same event on the app, selecting `Do not notify attendees`
     * Check your Gmail - you'll have a bounce back!

In summary, my stance on this is: the issue is not of OGCS's making, has been mitigated already and does not warrant further time spent on it. There are plenty of other areas in which to improve and progress OGCS, for which I already struggle to get enough time for.
